.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 lib
 es
+.idea
```

Comment:
Actually i'm not a fan of adding IDE specifics to project, since if you were to add files for every IDE/Tool to the ignore file, you'll be soon lost in a list files where you don't know what's relevant and what is not. It's also better to add `.idea` to your global `.gitignore` file since it will prevent you from committing `.idea` folder accidentally.